web/template/certificateprovider/confirm_your_details.gohtml

```diff
@@ -42,11 +42,9 @@
       <h2 class="govuk-heading-m govuk-!-margin-top-8">{{ tr .App "detailsTheDonorHasGivenAboutYou" }}</h2>
 
       {{ $detailsContent := "whatToDoIfAnyDetailsAreIncorrectCertificateProviderContentLay" }}
-      {{ $addressLabel := "address" }}
 
       {{ if .Lpa.CertificateProvider.Relationship.IsProfessionally }}
         {{ $detailsContent = "whatToDoIfAnyDetailsAreIncorrectCertificateProviderContentProfessional" }}
```

Comment:
I know this isn't a label, but I think we should do it for all switched content regardless of what it is

Reply:
I do like the idea of moving logic out of the templates so it can all be tested